apps/src/lib/levelbuilder/script-editor/UnitCard.jsx

```diff
@@ -98,14 +111,65 @@ class UnitCard extends Component {
    */
   serializeLesson = lesson => {
     let s = [];
-    let t = `stage '${escape(lesson.name)}'`;
+    let t = `lesson '${escape(lesson.name)}'`;
+    if (lesson.name) {
+      t += `, display_name: '${escape(lesson.name)}'`;
+    }
     if (lesson.lockable) {
       t += ', lockable: true';
     }
+    if (lesson.visible_after) {
+      t += ', visible_after: true';
+    }
     s.push(t);
+    if (lesson.levels) {
+      lesson.levels.forEach(level => {
+        s = s.concat(this.serializeLevel(level.ids[0], level));
+      });
+    }
+    s.push('');
     return s.join('\n');
   };
 
+  serializeLevel = (id, level, active) => {
```

Comment:
I am a bit lost, why is it that we need to serialize levels on the script edit gui page? that should only contain Units, Lesson Group and Lesson now, right?

Reply:
I will add comments around why levels are included here without being able to edit